android/ci_build_libs_cmake.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+set -e
+
+PLATFORM_NAME="android"
+
+if [ ! -d "$PLATFORM_NAME" ]; then
+    echo "start this script in the source root directory"
+    exit 1
+fi
+
+BUILD_DIR="$PWD/3rdParty/$PLATFORM_NAME"
+VCPKG_ROOT="$BUILD_DIR/vcpkg"
+VCPKG_PORTS="$PWD/3rdParty/vcpkg_ports"
+. $PLATFORM_NAME/common_ndk.sh
```

Comment:
Please rename to 'ndk_common' because we do not have common ndk but instead we have some common stuff for ndk, so this name will be more logically correct

Reply:
What name you want me to rename?

Reply:
`ndk_common` sound more reasonable, thank you.